### ISSUE TITLE
[BUG] Heading structure fails WCAG 2.2 AA - Skipped heading level

### ISSUE THREAD

**Issue by YuriDevAT** [EddieHubCommunity/BioDrop#9581]
### Has this bug been raised before?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Where did you find this bug?

production on biodrop.io

### Version of BioDrop (for example "v1.2.3")

v2.80.1

### Description

Checking the source code with WAVE and manually, it is shown that the heading structure skips one level.

**Example:** EddieJaoude profile https://www.biodrop.io/eddiejaoude

The first heading that appears is
`<h1 class="flex text-3xl font-bold gap-1">Eddie Jaoude | EddieHub<!-- --> <!-- -->(he/him)</h1>`

The next heading that appears is
`<h3 class="ml-2 mt-2 text-lg font-medium leading-6 dark:text-primary-low text-primary-high">Socials Primary</h3>`

`<h2></h2>` is missing. This is misleading for screen reader users.

### Screenshots

<img width="563" alt="Wave results for skipped heading level" src="https://github.com/EddieHubCommunity/BioDrop/assets/54622834/41c1d741-a0c4-4b35-b6ab-81dddb43ad70">

### Do you want to work on this issue?

Yes

### If you want to work on this issue...

I would turn all the `<h3>` of this page into a `<h2>`.
Since the heading tags have a styling set using tailwind, nothing changes in their appearance.

**Shoutout:** This problem is already prevented in the footer section through
`<h2 id="footer-heading" class="sr-only">Footer</h2>`

Good job 🚀

**Comment by SaraJaoude** [comment]
Thanks for raising! Unfortunately I cannot assign this to you as you already have one issue assigned (we only assign one issue per contributor).

**Comment by berryjen** [comment]
could i please give this issue a go? I will attempt my best >'<

**Comment by YuriDevAT** [comment]
And here goes my beautiful a11y issue 🥲 I am happy to switch tickets!

**Comment by MeenuyD** [comment]
I would like to work on this issue can you please assign it to me

**Comment by SudeeptaGiri** [comment]
I want to work on this issue. Can you Please assign me this Issue  ;-)

**Comment by Koushik1311** [comment]
I want to contribute to BioDrop can you assign this issue to me?

**Comment by SaraJaoude** [comment]
> could i please give this issue a go? I will attempt my best >'<

@berryjen I have assigned this to you

**Comment by berryjen** [comment]
sounds great Sara thank you! I will begin working on it :)
